src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/ExceptionsController.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Diagnostics.Monitoring.WebApi.Exceptions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Net.Http.Headers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Controllers
+{
+    [Route("")]
+    [ApiController]
+    [HostRestriction]
+    [Authorize(Policy = AuthConstants.PolicyName)]
+    [ProducesErrorResponseType(typeof(ValidationProblemDetails))]
+    [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
+    public sealed class ExceptionsController : ControllerBase
+    {
+        private readonly IExceptionsStore _exceptionsStore;
+        private readonly IInProcessFeatures _inProcessFeatures;
+        private readonly IExceptionsOperationFactory _operationFactory;
+
+        public ExceptionsController(IServiceProvider serviceProvider)
+        {
+            // The exceptions store for the default process
+            _exceptionsStore = serviceProvider.GetRequiredService<IExceptionsStore>();
+            _inProcessFeatures = serviceProvider.GetRequiredService<IInProcessFeatures>();
+            _operationFactory = serviceProvider.GetRequiredService<IExceptionsOperationFactory>();
+        }
+
+        /// <summary>
+        /// Gets the exceptions from the default process.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
```suggestion
```

Reply:
I noticed we don't have this on our other ones and wasn't sure if it was intentionally included here, feel free to disregard.